packages/legacy-workbench/package.json

```diff
@@ -38,37 +35,17 @@
         "babel-loader": "^8.1.0",
         "babel-polyfill": "^6.26.0",
         "babel-preset-es2015": "^6.24.1",
-        "clean-webpack-plugin": "^4.0.0",
-        "copy-webpack-plugin": "^12.0.2",
         "copyfiles": "^2.4.1",
-        "css-loader": "^3.3.2",
-        "css-minimizer-webpack-plugin": "^7.0.0",
-        "ejs-loader": "^0.5.0",
         "eslint": "^5.16.0",
         "eslint-config-google": "^0.13.0",
-        "expose-loader": "^0.7.5",
-        "extract-loader": "^5.1.0",
-        "file-loader": "^4.3.0",
-        "html-loader": "^0.5.5",
-        "html-webpack-plugin": "^5.6.0",
-        "ignore-loader": "^0.1.2",
         "is-obj": "^2.0.0",
         "less": "^3.10.3",
         "less-loader": "^5.0.0",
         "license-checker": "^25.0.1",
         "mini-css-extract-plugin": "^0.8.0",
```

Comment:
Check if this webpack plugin is still needed here

Reply:
Done